src/components/BMDashboard/Issue/Issue.jsx

```diff
@@ -0,0 +1,347 @@
+          <Row>
+            <Col>
+              <Label className="sub-title-text">{ISSUE_TYPE}</Label>
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <Input
+                className="issue-form-override-css"
+                type="select"
+                name="dropdown"
+                value={formData.dropdown}
+                onChange={e => handleDropdownChange(e.target.value)}
+              >
+                {dropdownOptions.map(option => (
+                  <option key={option} value={option}>
+                    {option}
+                  </option>
+                ))}
+              </Input>
+            </Col>
+          </Row>
+        </FormGroup>
+        <FormGroup>
+          <Row>
+            <Col>
+              <Label className="sub-title-text">{CONSEQUENCES_TITLE}</Label>
+            </Col>
+          </Row>
+          {checkboxOptions.map(pair => (
+            <Row key={pair} className="consequences-checkboxes">
+              {pair.map(option => (
+                <Col key={option}>
+                  <Label check>
+                    <Input
+                      type="checkbox"
+                      checked={formData.checkboxes.has(option) || false}
+                      onChange={() => handleCheckboxChange(option)}
+                    />
+                    {option}
+                  </Label>
+                  {option === 'Other' && (
+                    <Input
+                      className="issue-form-override-css"
+                      type="textarea"
+                      rows="1"
+                      placeholder="If other is selected, please specify."
+                      value={formData.other}
+                      onChange={e => handleOtherInputChange(e, option)}
+                    />
+                  )}
+                </Col>
+              ))}
+            </Row>
+          ))}
+        </FormGroup>
+        <FormGroup>
+          <Row>
+            <Col>
+              <Label className="sub-title-text">{RESOLVED}</Label>
+            </Col>
+          </Row>
+          <Row className="issue-radio-buttons">
+            <Col className="issue-radio-buttons">
+              <Label check>
+                <Input
+                  type="radio"
+                  name="radioOption"
+                  onChange={() => setFormData({ ...formData, resolved: 'Yes' })}
+                />
+                Yes
+              </Label>
+            </Col>
+            <Col className="issue-radio-buttons">
+              <Label check>
+                <Input
+                  type="radio"
+                  name="radioOption"
+                  onChange={() => setFormData({ ...formData, resolved: 'No' })}
+                />
+                No
+              </Label>
+            </Col>
+          </Row>
+        </FormGroup>
+        <FormGroup>
+          <Row>
+            <Col>
+              <Label for="description" className="sub-title-text">
```

Comment:
There is console error for label element as shown below:
![image](https://github.com/user-attachments/assets/c9d0b7c1-6fe5-4d8c-9b5c-1384e9eeac79)